app/services/reporting/account_deletion_rate_report.rb

```diff
@@ -6,6 +6,18 @@ def initialize(report_date = Time.zone.today)
       @report_date = report_date
     end
 
+    def account_deletion_report_title
+      'Account deletion rate (last 30 days)'
+    end
+
+    def account_deletion_report_metadata
+      {
+        title: account_deletion_report_title,
+        float_as_percent: true,
+        precision: 4,
+      }
+    end
```

Comment:
Arguably, this stuff _belongs_ here anyway. The AccountDeletionRateReport should know how it wants its data to be formatted.

Reply:
should we make this a common struct like you were talking about earlier and have each report class return a the full struct? instead of having MKMR glue the pieces together?

Reply:
I think so. I wonder about how much we want to tackle all at once, though. Maybe that's a follow-on?